tests/gha-run-k8s-common.sh

```diff
@@ -364,3 +364,14 @@ function delete_test_cluster_namespace() {
 	set_default_cluster_namespace
 	kubectl delete namespace "${TEST_CLUSTER_NAMESPACE}"
 }
+
+function delete_test_scripts(){
```

Comment:
I would rather call this something like `delete_test_runners()` 🙂

Reply:
Done.